likely to ever happen.
      MaybeRaiseExceptionFromStatus(
          tensorflow::errors::Internal(
             